index.js

```diff
@@ -10,11 +10,8 @@ module.exports = function(out, options) {
 
   var onFile = function(file) {
     files.push(file);
-    var path = file.path;
-    if (!options.absolute) {
-      path = path.replace(new RegExp('^' + process.cwd() + '/'), '');
-    }
-    filePaths.push(path);
+    var path = options.absolute ? file.path : file.relative;
```

Comment:
The idea was to take care of the `base` option of `gulp.src` and to build accordingly the path.
The results are: 
- no base: `test.file`
- `./` : `test/test.file`
- `./test/` : `test.file`